README.md

```diff
@@ -26,6 +26,8 @@ Before any Pull Request is landed, sufficient time should be given to receive in
 
 If it becomes apparent that the changes proposed by a given Pull Request: (a) have significant impact on the project as a whole; (b) are inherently controversial; or (c) have failed to reach consensus amongst Collaborators, the Pull Request can be elevated for review by either a specific Working Group (by attaching a WG specific tag to the PR) or the TSC (by attaching the *tsc-agenda* tag to the PR). Pull Requests that have been flagged for TSC or Working Group review must not be landed until the TSC or Working Group has had sufficient opportunity to review the issue and render a decision. A Working Group may choose to defer the issue to the TSC.
 
+Pull Requests that require an increase in the Major version must be elevated for review by the TSC.
```

Comment:
Maybe a little more language about this would be helpful, usually it isn't worthwhile to raise anything to the TC (at least in a meeting) immediately.